.ado/jobs/e2e-test.yml

```diff
@@ -56,10 +56,25 @@ jobs:
         condition: false # Must be manually enabled, since it causes a 5x perf reduction that causes test instability
 
       - task: CmdLine@2
-        displayName: run-windows
+        displayName: run-windows - PR
         inputs:
-          script: yarn windows --no-launch --no-packager --arch $(BuildPlatform) --release --logging --buildLogDirectory $(BuildLogDirectory) --msbuildprops BaseIntDir=$(BaseIntDir)
+          script: yarn windows --arch $(BuildPlatform) --logging --buildLogDirectory $(BuildLogDirectory) --msbuildprops BaseIntDir=$(BaseIntDir) AppxPackageSigningEnabled=false
           workingDirectory: packages/e2e-test-app
+        condition: eq('${{ parameters.BuildEnvironment }}', 'PullRequest')
+
+      - task: DownloadSecureFile@1
+        name: signingCert
+        displayName: 'Download CA certificate'
+        inputs:
+          secureFile: 'reactUWPTestApp_Key.pfx'
+        condition: eq('${{ parameters.BuildEnvironment }}', 'Continuous')
+
+      - task: CmdLine@2
+        displayName: run-windows - CI
+        inputs:
+          script: yarn windows --no-launch --no-packager --arch $(BuildPlatform) --release --logging --buildLogDirectory $(BuildLogDirectory) --msbuildprops BaseIntDir=$(BaseIntDir) PackageCertificateKeyFile="$(signingCert.secureFilePath)"
```

Comment:
Should we parameterize the arguments in this case? 
Similar to: `SkipNpmPublishArgs` in publish.yml